recipes/parsnp/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+./autogen.sh
+./configure --prefix=${PREFIX}
+make
+sed -i.bak '1i#!/usr/bin/env python\n' Parsnp.py
+sed -i.bak 's/sys\.stderr\.write(\ BOLDME/sys\.stdout\.write(\ BOLDME/' Parsnp.py
```

Comment:
Can you give any information as to why changing the program's behavior this way is necessary?

Reply:
For the second sed: whenever the program used to be called, the version and the link to the documentation would be sent to stderr, prompting any test to fail for instance.

For the first one: I thought this was standard procedure for directing the system to use python for interpreting the script files.

Reply:
But your change will alter the program's output (on `stdout`). I don't believe this should be done.
The `stderr` stream is commonly used not only for outputting errors but also logging information. Hence I believe the original usage is totally legitimate. If this caused any tests to fail, then those tests are likely broken

Reply:
Sorry you are right I confused to things at the same time. It was calling parsnp without any arguments that throws an error code, I'll remove the second sed.

Reply:
Great, thanks!